sctp/src/queue/pending_queue.rs

```diff
@@ -25,16 +25,52 @@ impl PendingQueue {
         PendingQueue::default()
     }
 
+    /// Appends a chunk to the back of the pending queue.
     pub(crate) async fn push(&self, c: ChunkPayloadData) {
```

Comment:
Do we need to keep push at all?

Reply:
we're using it to send "reset" chunk. although it's possible to wrap it in a vector.

Reply:
I guess not having `push` would encourage us to not make a similar mistake to this in the future. I'll leave it with you to make a decision